background.ts

```diff
@@ -456,7 +456,9 @@ Electron.ipcMain.handle('service-forward', async(event, service, state) => {
  * (not installed, but can be), or null (install unavailable, e.g. because a
  * different executable already exists).
  * @param {string} name The name of the executable, e.g. "kubectl", "helm".
- * @returns {boolean?} The state of the installable binary.
+ * @returns [{boolean?} state, {string} failureReason]
```

Comment:
We should be able to put the type info into the code:
```ts
async function refreshInstallState(name: string): Promise<[boolean | null, string | null]> {
```
And if we don't, the `{}` is delimiting types in JSDoc:
```suggestion
 * @returns {[boolean | null, string | null]}
```

Reply:
done